### PR TITLE
feat: add paging flags for list commands that support paging

### PR DESCRIPTION
**Pull request by kangasta**
This also increases the default page size from 10 to 100.

### CODE REVIEW

File: internal/paging/paging.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+)
+
+type PageParameters struct {
+	size   int
+	number int
+}
+
+func (pp *PageParameters) ConfigureFlags(fs *pflag.FlagSet) {
+	fs.IntVar(&pp.size, "size", 100, "Number of entries to receive at most.")
+	fs.IntVar(&pp.number, "page", 0, "Page number to calculate first item to receive. Page numbers start from `1`.")
```
Comment (kangasta) [inline internal/paging/paging.go:15]: These could also be `--page-size` and `--page-number` to be more descriptive, but those are quite verbose 🤔

File: internal/paging/paging.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package paging
```
Comment (kangasta) [inline internal/paging/paging.go:1]: Could also be `list` package if we add label filtering to the same package 🤔 Easy to rename later as well.

Comment (villevsv-upcloud) [reply internal/paging/paging.go:1]: Renaming later, if needed, sounds reasonable.

File: internal/paging/paging.go (hunk 1 above)
Comment (villevsv-upcloud) [reply internal/paging/paging.go:15]: As they naturally are available only on `list` commands, these shorter ones seem fine to me.

**Review by villevsv-upcloud — APPROVED** [review]

File: internal/paging/paging.go (hunk 1 above)
Comment (kangasta) [reply internal/paging/paging.go:15]: I was also thinking between `--size` and `--limit`. Go SDK uses size, but there that is used as field of `Page` struct. Maybe limit would clearer without that context 🤔

Comment (kangasta) [reply internal/paging/paging.go:15]: Changed `--size` → `--limit`

**Pull request merged**